keybase/platform_windows.go

```diff
@@ -125,6 +139,37 @@ func CheckRegistryUninstallCode(productID string, log Log) bool {
 	return false
 }
 
+// Read all the runonce subkeys and find the one with "Keybase" in the name.
+func checkRebootPending(wow64 bool) (result bool) {
+	var access uint32 = registry.ENUMERATE_SUB_KEYS | registry.QUERY_VALUE
+	if wow64 {
+		access = access | registry.WOW64_32KEY
+	}
+
+	k, err := registry.OpenKey(registry.CURRENT_USER, "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\RunOnce", access)
+	if err != nil {
+		fmt.Printf("Error %s opening RunOnce subkeys\n", err.Error())
```

Comment:
Don't std out, use logging..

And log the error at the end... e.g. `log.Errorf("Error opening RunOnce subkeys: %s", err)`.

And don't need `\n`.